third_party/docfx/templates/devsite/partials/uref/parameters.tmpl.partial

```diff
@@ -1,7 +1,7 @@
 {{!Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.}}
 
 {{#properties.0}}
-<h6 class="parameters">Properties</h6>
+<strong>Properties</strong>

```

Comment:
I don't think this should just be `strong`?

Reply:
The header above it would likely be `<strong>{{__global.parameters}}</strong>`, if not then `h3` above, so it would have to be h4 at best. I think it's fine to keep it as strong.